app/src/main/res/values-uk/strings.xml

```diff
@@ -255,7 +255,7 @@
         • Змінюйте видимість файлів, щоб приховати фото і відео
         • Створюйте налаштовуване слайд-шоу з ваших файлів
         • Переглядайте детальну інформацію про ваші файли (роздільна здатність, записи EXIF тощо)
-        • Simple Gallery Pro є додатком з відкритим програмним кодом
+        • Simple Gallery Pro є додатком з відкритим джерельним кодом
```

Comment:
The word 'джерело' has several meanings: see https://uk.wiktionary.org/wiki/джерело paragraph 'Значення'.
I see that other translators try to avoid the usage of 'програма', 'програмний' and use the word 'джерело' to indicate that this is the initial source, the beginning of everything. Though I don't perfectly like this translation, I believe it's the closest meaning.